### ISSUE TITLE
Close icon should be a button 🙅‍♀️

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#265]
The close button doesn't have a label. It should also be a button instead of a link.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592509-b1c821dc-9643-11e8-8c76-2cd3f9c6a318.png">

**Comment by arshadkazmi42** [comment]
I can take this up, if its still available

**Comment by soleo** [comment]
<img width="1024" alt="screen shot 2018-10-06 at 12 09 04 pm" src="https://user-images.githubusercontent.com/635858/46573882-b5dcf400-c960-11e8-8bdf-546558a0a550.png">
 Looks like this has been fixed in current master branch

**Comment by bookofbash** [comment]
Should this issue be closed?

**Comment by jonathanKingston** [comment]
Closing as role button and label was added here.
